modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -2,6 +2,8 @@
 [id="cluster-logging-collector-log-forward-loki_{context}"]
 = Forwarding logs to Loki
 
+//Eng-Feedback-Req: Can any of this be recycled to meet doc needs for Loki?
+
```

Comment:
Probably this section will remain to document connecting to an external Loki. There are some differences in authentication when connecting to a local Loki.  As we talked about before, my preference would be to have a quick example of the most common way to connect to an external loki, a quick section explaining the (mostly automatic) connection to a local default loki, both linked to an independent, detailed reference section explaining _all_ of the options for a Loki output, so the user can handle their specific requirements.